src/test/pybind/test_rados.py

```diff
@@ -240,7 +241,7 @@ def test_get_pool_base_tier(self):
 
     def test_get_fsid(self):
         fsid = self.rados.get_fsid()
-        eq(len(fsid), 36)
+        assert re.compile('[0-9a-f\-]{36}', re.I).match(fsid)
```

Comment:
nit,

```py
assert re.match('[0-9a-f\-]{36}', fsid, re.I)
```

would suffice. there is not much value to compile a throw-away re object.